foundry/foundry.py

```diff
@@ -47,10 +44,10 @@ class Foundry(FoundryMetadata):
     index = ""
     auths: Any
 
-    xtract_tokens: Any
-
     def __init__(
-            self, no_browser=False, no_local_server=False, index="mdf", authorizers=None, **data
+            self, no_browser=False, no_local_server=False, index="mdf", authorizers=None,
+            globus=True, verbose=False, interval=10,
```

Comment:
I thought it might make sense to keep these here, as I was thinking of them as 'session' specific. If we implement the cache as discussed, there won't be any specific call to download the data (instead just accessing it via something like `dataset.as_dataframe()`), so the cache will have to be ready to do the downloading if necessary. Having those defined during instantiation of the foundry object allows them to be session-specific variables, ready to go if/when needed. Happy to chat about this if you have different thoughts though?

Reply:
that sounds fine to me, thanks for the reasoning!